Automattic/woocommerce-payments
# Remove "new checkout experience" setting + Enable using WooPay and UPE at the same time

Fixes #4724

#### Changes proposed in this Pull Request

Remove "new checkout experience" setting

This PR removes the need to enable UPE and any banner, button, or message related to disabling it or referencing "Early access". This was done because now UPE methods are separate methods on checkout, and the credit card element being used is the legacy one, which works with WooPay, so there's no need to ever disable UPE entirely, as disabling each method individually would achieve the same goal.

#### Testing instructions

1. Before checking out this branch, disable UPE for testing.
1. Go to WooCommerce Payments settings and notice there's no UPE banner, and all payment methods are displayed.
1. Enable some of these UPE methods and WooPay.
1. Change the store currency to EUR.
1. Make a purchase using a UPE method and ensure it works.
1. Make a purchase using WooPay and ensure it works.

-------------------


**Post merge**
